compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -3337,8 +3337,12 @@ boolean validateErrorVariable(BLangErrorVariable errorVariable, SymbolEnv env) {
                 defineMemberNode(errorVariable.message, env, symTable.stringType);
             }
 
-            BLangVariable cause = errorVariable.cause;
-            if (cause != null) {
+            BLangVariable errorCause = errorVariable.cause;
+            if (errorCause != null) {
+                if (errorCause.getKind() == NodeKind.VARIABLE && names.fromIdNode(((BLangSimpleVariable) errorCause).name) == Names.IGNORE) {
```

Comment:
Does checkstyle pass for this? Isn't the line longer than 120?

Reply:
[7227649](https://github.com/ballerina-platform/ballerina-lang/pull/39691/commits/72276494ccc1e153b3e19a7fc05f8884f8a44371)